src/me/drton/jmavsim/MAVLinkHILSystem.java

```diff
@@ -257,6 +257,28 @@ public void update(long t) {
             msg_system_time.set("time_boot_ms", tu/1000);
             sendMessage(msg_system_time);
         }
+        
+        // Optic flow
+        if (sensors.isFlowUpdated()) {
+            FlowData flowData = sensors.getFlowData();
+            if (flowData != null && flowData.integrated_flow != null && flowData.integrated_gyro != null) {
+            //if (true) {
```

Comment:
This `if (true)` can probably be removed?

And what about a command line argument to enable flow?

Reply:
Yes an argument would be nice, does it exist for other sensors?

Reply:
I don't think so but it exists for other runtime settings.